File: src/ProjectR/content/scripts/spells/SecondWind.lua
function GetTargetType()
    return Single
end

function GetName()
  return "Second Wind"
end

function GetDescription()
    return "Heal target with the power of the wind."
end

function IsSupportSpell()
    return true
end

function GetMPCost()
    return .18
end

function GetDelay()
  return .5
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return { WND }
end

function SpellEffect()
  local healing = 1.8 * aMD * (aWND/100)
  d:Heal(healing)
end

File: src/ProjectR.Scripting/content/scripts/spells/SecondWind.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class SecondWind : SpellScriptBase
    {
        public override string Name { get { return "Second Wind"; } }
        public override string Description { get { return "Heal target with the power of the wind."; } }

        public override TargetType TargetType { get { return TargetType.Single; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { EleMastery.WND,  }; } }
        public override SpellType SpellType { get { return SpellType.Pure; } }
        public override bool IsSupportSpell { get { return true; } }

        public override double MPCost { get { return 18; } }
        public override double Delay { get { return .5; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            var healing = 1.8 * aMD * (aWND / 100);
            Heal(healing);
        }
    }
}